data/categories.txt

```diff
@@ -1911,7 +1911,7 @@ nl:3^Metro
 fi:3^Metro
 fr:3^Métro
 de:2^U-Bahn|Metro|ÖPNV
-hu:3^Földalatti|3metro
+hu:3^Metró|Földalatti|3metro
```

Comment:
Fill free to drop the last one: diacritics are ignored when looking for synonyms.

Reply:
OK, I will drop it (or you can do as well).